### PR TITLE
✨ Added cmdlets  to manage web server type

### PR DESCRIPTION
**Pull request by smholvoet**
Added `Get-D365WebServerType` and  `Set-D365WebServerType` cmdlets which allow you to quickly change the web server to either IIS or IIS Express.

Both cmdlets are nearly identical to the existing `Get-D365DefaultModelForNewProjects` / `Set-D365DefaultModelForNewProjects` cmdlets which also replace the value of a specific XML node. Only thing I'm not quite sure of is the location of the `DynamicsDevConfig.xml` file. It will currently look for this file in `K:\AosService\PackagesLocalDirectory\bin\`, which should be the default service drive for most dev boxes...

- 💪 Fixes d365collaborative/d365fo.tools#571.
- ✔ added documentation
- ✔ added tests

### CODE REVIEW

File: d365fo.tools/functions/set-d365webservertype.ps1 [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseSingularNouns", "")]
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, ValueFromPipeline = $true)]
+        [ValidateSet('IIS', 'IISExpress')]
+        [string] $RuntimeHostType
+    )
+
+    begin {
+        $filePath = "K:\AosService\PackagesLocalDirectory\bin\DynamicsDevConfig.xml"
```
Comment (smholvoet) [inline d365fo.tools/functions/set-d365webservertype.ps1:45]: Assuming `K:` is the default service drive for most dev boxes.

**Comment by Splaxi** [comment]
I would recommend that you considered implementing matching backup / restore cmdlets, just like:
https://github.com/d365collaborative/d365fo.tools/blob/development/d365fo.tools/functions/backup-d365webconfig.ps1
https://github.com/d365collaborative/d365fo.tools/blob/development/d365fo.tools/functions/restore-d365webconfig.ps1

If you really want to support that people will be able to "easily" recover the DynamicsDevConfig.xml file again.

**Comment by Splaxi** [comment]
Awesome work! 🤘

File: d365fo.tools/functions/get-d365webservertype.ps1 [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        Author: Sander Holvoet (@smholvoet)
+   
+#>
+
+function Get-D365WebServerType {
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseSingularNouns", "")]
+    [CmdletBinding()]
+    param ()
+
+    $filePath = "K:\AosService\PackagesLocalDirectory\bin\DynamicsDevConfig.xml"
```
Comment (FH-Inway) [inline d365fo.tools/functions/get-d365webservertype.ps1:28]: You can use the ```$Script:ServiceDrive``` variable to get the drive of the AosService folder. See [Add-D365WindowsDefenderRules](https://github.com/d365collaborative/d365fo.tools/blob/c34e0fd70311f7b368c618f0a54c29a917428d53/d365fo.tools/functions/add-d365windowsdefenderrules.ps1) for an example.

Comment (Splaxi) [reply d365fo.tools/functions/get-d365webservertype.ps1:28]: We will be using the `$Script:PackageDirectory` variable, which is more common across the module.

**Pull request merged**